src/overlays/actors/ovl_En_Door/z_en_door.h

```diff
@@ -4,14 +4,57 @@
 #include <ultra64.h>
 #include <global.h>
 
+/**
+ * Actor Parameters
+ *
+ * |                  |       |             |
+ * | Transition Index | Type  | Double Door | Switch Flag OR Text Id - 0x0200
+ * |------------------|-------|-------------|---------------------------------
+ * | 0 0 0 0 0 0      | 0 0 0 | 0           | 0 0 0 0 0 0
+ * | 6                | 3     | 1           | 6
+ * |
+ *
+ * Transition Index     1111110000000000    Set by the actor engine when the door is spawned
```

Comment:
it's or `|`'d, not set, so it being 0 initially matters, idk if that's worth pressing on though

Reply:
The instance space is zero'd out with a call to `Lib_MemSet` on spawn so this isn't worth specifying in particular.

Reply:
I meant if one was to use this documentation for building a transition actors list in a scene header

Still, what's the point of `Lib_MemSet`? the actor.params is set afterwards anyway?

Reply:
MemSet ensures no data that was previously allocated in that space will cause it to be nonzero. This concern seems like the kind of thing that will be covered when macros for getting and setting actor params are introduced in the future, perhaps at that stage a note can be left on the setter for the transition index that it is for internal use only. That should cover the issue entirely as once they are introduced no actor params should be edited without reference to these macros.

Reply:
I think MemSet is completely irrelevant for what I'm talking about
https://github.com/zeldaret/oot/blob/14191307e96cc915fa5a67704eba16421cf4deb2/src/code/z_actor.c#L2818
I was mistaken it's a `+` not a `|` but the issue is unchanged

Whatever. macros should indeed eliminate the concern